flags/flags.go

```diff
@@ -145,6 +145,19 @@ as "300ms", "-1.5h" or "2h45m". Valid time units are "ns", "us" (or "µs"), "ms"
 		Usage: "The provisioner <name> to use.",
 	}
 
+	// AdminProvisioner is a cli.Flag used to pass the CA Admin provisioner to use.
+	AdminProvisioner = cli.StringFlag{
+		Name:  "admin-provisioner,admin-issuer",
+		Usage: "The provisioner <name> to use for generating admin credentials.",
+	}
+
+	// AdminSubject is a cli.Flag used to pass the admin subject to use when generating
+	// admin credentials.
+	AdminSubject = cli.StringFlag{
+		Name:  "admin-subject,admin-name",
+		Usage: "The admin <subject> to use for generating admin credentials.",
+	}
```

Comment:
I would remove the aliases, but it's ok as it is.

Reply:
mike asked for `admin-name` and I left `admin-issuer` so it would match the alias we have for `provisioner`.